docs/pages/installation.md

```diff
@@ -248,6 +270,12 @@ Here's what comes in the package.
   - `js/`: Concatenated JavaScript files. Includes minified and unminified files.
     - `plugins/`: Standalone JavaScript plugins.
 
+Once installed with one of the above commands, Foundation has been downloaded via git and installed into the `assets/` directory.
+
+If you are using Foundation Sass, you'll `@import` Foundation from the `scss/` folder. You will also link to the `js/` folder where all the uncompressed JS files live.
```

Comment:
These are two separate concepts.  The Sass statement is correct... the JS folder piece should be for if you're using a JS build system and integrating this into an ES2016 project.